### PR TITLE
Disable Big Sur check and specify DevTools version

### PR DESCRIPTION
**Pull request by helin24**

### CODE REVIEW

**Review by kenzieschmoll — APPROVED** [review]

File: src/io/flutter/run/daemon/DevToolsService.java [hunk 1]
```diff
@@ -233,7 +234,7 @@ public void projectClosing(@NotNull Project project) {
   }
 
   private CompletableFuture<Boolean> pubActivateDevTools(FlutterSdk sdk) {
-    final FlutterCommand command = sdk.flutterPub(null, "global", "activate", "devtools");
+    final FlutterCommand command = sdk.flutterPub(null, "global", "activate", "devtools", "2.0.0-dev.1");
```
Comment (devoncarew) [inline src/io/flutter/run/daemon/DevToolsService.java:237]: Do we indent to stay fixed at this specific version (`2.0.0-dev.1`) for a while? If so, perhaps add a comment about why.

**Comment by kenzieschmoll** [comment]
Is this just for the test release? If so, can these changes just be local to the build?

**Comment by stevemessick** [comment]
I restarted Kokoro. It failed with some bizarre error.

**Comment by stevemessick** [comment]
Kokoro seems flakey. The unit tests are finished but the job did not terminate.

**Comment by helin24** [comment]
To be clear I only meant to put up this branch so Steve could put changes into the test release! Sorry about the confusion.

**Pull request closed without merging**